### DISCUSSION TITLE
Mouse pointer is a blue circle

### DISCUSSION THREAD

**Discussion by Glenn1963** [rustdesk/rustdesk#4945] [Q&A] [answered]
On Windows desktop v1.2.0, my mouse pointer / cursor is a blue circle on any remote desktop. This wasn't always the case, but I'm wondering if it is because now I have a touch-screen attached to the local system?

I can't see any way to override this behavior? I would like the regular pointer.

(I couldn't find a way to capture the screen with pointer, so here's a photo)
![image](https://github.com/rustdesk/rustdesk/assets/17497231/f719f336-5f11-4e02-b268-4690201d17fc)

Thanks for any thoughts...

**Comment by joernv** [comment]
For me, the mouse cursor was also displayed as a blue circle. The reason for this was that no mouse was connected to the remote-controlled PC. After connecting a mouse, the normal cursor was displayed again.

**Reply by Glenn1963** [reply, marked as answer]
Ah, cool, thanks for the tip :) The thought never occurred to me.

Looks like you can also use the Ease of Access setting 'Control your mouse with a keypad' to create a mouse. 
![image](https://github.com/rustdesk/rustdesk/assets/17497231/df89cdcb-2e60-49b9-88ce-b9c21de4e120)

This did work for me when I tried it but I don't have any long-term test yet.

Appreciate the heads-up!

**Reply by herbertyugh** [reply]
Same problem and same resaon here, and thanks for you tips. And my solution is install LogitechG HUB version 2021.11.1775 (the newest version doesn't support virtual mouse)
![image](https://github.com/rustdesk/rustdesk/assets/25643687/8a4edf5d-5538-4950-af1c-a8edceea3192)

**Comment by kvalle22** [comment]
On win 11 it's called mouse keys 🤷‍♂️
![image](https://github.com/user-attachments/assets/1e9fc77b-0606-4736-8a3a-a9f0d28d77bd)
![image](https://github.com/user-attachments/assets/6915a2cf-127c-4379-8c53-0805894dcb51)

**Comment by WinAdmCz** [comment]
My observations: when the remote computer has no mouse physically attached, then you get mouse pointer as blue circle. The same behaviour in Win10 and Win11. Using "Mouse keys" is so the logical workaround.

**Comment by outback20** [comment]
My remote cursor shows as a blue circle even though I have a wired USB mouse attached (via USB hub). I found that when I open the old Outlook desktop app (Microsoft 365), the remote cursor changes to its correct shape and color. The shape/color remain for the rest of the Rustdesk session after minimizing Outlook. Windows 11. None of the Rustdesk "remote cursor" options are set.

**Comment by DcryptedCA** [comment]
This keeps happening to me on Win11/Win11. I also have a NanoKVM on the PC though, but the cursor goes to the blue circle, sometimes arrows, and it can be reset by turning ON mouse keys and then back off. This only fixes it for the session.
